phing/tasks/setup.xml

```diff
@@ -229,6 +229,10 @@
         <drush command="config-import" assume="yes" alias="${drush.alias}" passthru="false">
           <param>${cm.core.key}</param>
         </drush>
+        <!-- Sometimes config-import has to be run twice to catch everything. Can be removed once the core configuration system is more stable. -->
```

Comment:
I would but I haven't been able to isolate all of the different ways this can happen... I think this workaround will be necessary for a while until the whole config subsystem is generally more mature.